TF Testing app (#2086)
saucelabs/sauce-docs: docs/test-fairy/testing-an-app/feedback.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

User/Tester's feedback can contain vital and highly relevant information when testing an app.

It can improve your app user experience and make it easier for your testers to communicate with you their thoughts on how to make your app better.

> ### Using in-app feedback as is

Author: ```suggestion
### Using In-app Feedback as is
```

> ### Using In-app Feedback as is

Reviewer: ```suggestion
### Using In-app Feedback
```

TestFairy provides an easy way to collect this feedback.

If you [added the TestFairy SDK](https://docs.testfairy.com/SDK/Adding_The_SDK_To_Your_App.html) to your app, then all you need to do is enable the **In-App Bug Reporting** feature in your build settings in the TestFairy dashboard, and you can start collection feedbacks from your users with `"shake to report"`:

<img src={useBaseUrl('/img/test-fairy/testing-an-app/enable_feedback.png')} alt="In-app bug reporting"/>

Users or testers will be prompted to report feedback when they shake their devices.

This feedback will be added to the existing app session they are currently running.

> All feedback includes a screenshot, device information, submitter email, and text comments added. The feedback is added to the event timeline so you can easily find it.

Author: ```suggestion
All feedback includes a screenshot, device information, submitter email, and text comments added. The feedback is added to the event timeline so you can find it without difficulty.
```

> ### Customizing In-app feedback

Author: ```suggestion
### Customizing In-app Feedback
```
